fastdeploy/backends/lite/lite_backend.cc

```diff
@@ -43,24 +43,34 @@ void LiteBackend::BuildOption(const LiteBackendOption& option) {
   option_ = option;
   std::vector<paddle::lite_api::Place> valid_places;
   if (option_.enable_int8) {
-    valid_places.push_back(
+    if(option_.enable_xpu){
+      valid_places.push_back(
+          paddle::lite_api::Place{TARGET(kXPU), PRECISION(kInt8)});
+    } else {
+      valid_places.push_back(
         paddle::lite_api::Place{TARGET(kARM), PRECISION(kInt8)});
+    }
     FDINFO << "Lite::Backend enable_int8 option is ON ! Lite::Backend will "
            << "inference with int8 precision!" << std::endl;    
   }
   if (option_.enable_fp16) {
-    paddle::lite_api::MobileConfig check_fp16_config;
-    // Determine whether the device supports the FP16
-    // instruction set (or whether it is an arm device
-    // of the armv8.2 architecture)
-    supported_fp16_ = check_fp16_config.check_fp16_valid();
-    if (supported_fp16_) {
+    if(option_.enable_xpu){
       valid_places.push_back(
-          paddle::lite_api::Place{TARGET(kARM), PRECISION(kFP16)});
-      FDINFO << "Your device is supported fp16 ! Lite::Backend will "
-             << "inference with fp16 precision!" << std::endl;    
+          paddle::lite_api::Place{TARGET(kXPU), PRECISION(kFP16)});
     } else {
-      FDWARNING << "This device is not supported fp16, will skip fp16 option.";
+      paddle::lite_api::MobileConfig check_fp16_config;
+      // Determine whether the device supports the FP16
+      // instruction set (or whether it is an arm device
+      // of the armv8.2 architecture)
+      supported_fp16_ = check_fp16_config.check_fp16_valid();
+      if (supported_fp16_) {
+        valid_places.push_back(
+            paddle::lite_api::Place{TARGET(kARM), PRECISION(kFP16)});
+        FDINFO << "Your device is supported fp16 ! Lite::Backend will "
+              << "inference with fp16 precision!" << std::endl;    
+      } else {
+        FDWARNING << "This device is not supported fp16, will skip fp16 option.";
+      }
```

Comment:
FDWARNING << "The device doesn't support FP16, will fallback to FP32." << std::endl;